src/views/product-tutorials-view/server.ts

```diff
@@ -1,26 +1,35 @@
+import { ProductData } from 'types/products'
 import { getAllCollections } from 'lib/learn-client/api/collection'
 import { getProduct } from 'lib/learn-client/api/product'
 import {
   Collection as ClientCollection,
   ProductOption,
-  Product as ClientProduct,
 } from 'lib/learn-client/types'
 import { stripUndefinedProperties } from 'lib/strip-undefined-props'
 import { filterCollections } from './helpers'
 
 export interface ProductTutorialsPageProps {
   collections: ClientCollection[]
-  product: ClientProduct
+  product: ProductData
```

Comment:
Since we're also passing product from the Learn API, we'll need to extend this to include the `ClientProduct`